robotbase/src/main/java/frc/robot/subsystems/ShooterPivot.java

```diff
@@ -64,26 +69,74 @@ public void stop() {
 
   public void resetEncoder() {
     m_pivotMotor.getEncoder().setPosition(0);
-    m_targetRotations = 0;
+    m_targetSetpoint = 0;
   }
 
   public double getPivotRotations() {
     return m_pivotMotor.getEncoder().getPosition();
   }
 
-  public double getAngle() {
+  public double getPosition() {
     return m_absoluteEncoder.getPosition();
   }
 
-  public boolean isPivotAtRotations() {
+  public boolean isPivotAtSetpoint() {
     return Utility.isWithinTolerance(
-        getAngle(), m_targetRotations, SHOOTER_PIVOT.POSITION_ALLOWED_ERROR);
+        getPosition(), m_targetSetpoint, SHOOTER_PIVOT.POSITION_ALLOWED_ERROR);
+  }
+
+  public boolean hasTarget() {
```

Comment:
I would make this method private, it is just a short hand version of getting something that can be gotten anywhere.

Reply:
I would make the same change in the Shooter subsystem